karma.conf.saucelabs.js

```diff
@@ -23,12 +23,13 @@ var browsers = {
 
 module.exports = function (config) {
   karmaBaseConf.plugins.push(require('karma-sauce-launcher'))
+  karmaBaseConf.plugins.push(require('karma-firefox-launcher'))
```

Comment:
This is wrong, should be in the `karma.confg.base.js`

--> https://github.com/contentful/contentful-management.js/pull/127/files#diff-e3ca3f9c4ef50d2071076da33038a6fd
--> https://github.com/contentful/contentful-management.js/pull/127/files#diff-41f2621a7581f6a00915e53d89371399